clip.cpp

```diff
@@ -1120,7 +1122,46 @@ struct tile_feature {
 	size_t seq = 0;
 };
 
-static void feature_out(std::vector<tile_feature> const &features, mvt_layer &outlayer, std::set<std::string> const &keep, std::unordered_map<std::string, attribute_op> const &attribute_accum, std::shared_ptr<std::string> const &tile_stringpool) {
+static void add_mean(mvt_feature &feature, mvt_layer &layer, std::string const &accumulate_numeric) {
+	std::string accumulate_numeric_colon = accumulate_numeric + ":";
+
+	std::unordered_map<std::string, size_t> attributes;
+	for (size_t i = 0; i + 1 < feature.tags.size(); i += 2) {
+		std::string const &key = layer.keys[feature.tags[i]];
+		if (starts_with(key, accumulate_numeric_colon)) {
+			attributes.emplace(key, i);
+		}
+	}
+
+	for (size_t i = 0; i + 1 < feature.tags.size(); i += 2) {
+		std::string accumulate_numeric_sum_colon = accumulate_numeric + ":sum:";
+
```

Comment:
Let's test that properties like `felt:h3_index` don't get accumulated

Reply:
Verified that they make it through into the point tiles but are not accumulated.